### PR TITLE
FOR REVIEW: RPIP-36: Committee Membership Record

### PR DESCRIPTION
**Pull request by LongForWisdom**
Not a huge amount to say on this one. More or less self-explanatory.

I'm happy to modify structure / formatting if anyone makes a good argument for some improvement. 

My staging branch is currently busy. Its fine, I promise. Just trust me bro.

### CODE REVIEW

**Comment by LongForWisdom** [comment]
Note from Patrick: Add something re future committees. Intention is to have them in this RPIP. Editors can judge what a committee is, probably.

**Comment by LongForWisdom** [comment]
Update to RPIP-36 before commit

**Comment by LongForWisdom** [comment]
Note, now update with the latest GMC members.

File: RPIPs/RPIP-36.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+| Incentives Management Committee (IMC) | To support low slippage exchange between ETH and rETH                  | 2022-08-25       | [RPIP-20](RPIP-20), [RPIP-10](RPIP-10) | [Mainnet](https://app.safe.global/home?safe=eth:0xb867EA3bBC909954d737019FEf5AB25dFDb38CB9), [Arbitrum](https://app.safe.global/home?safe=arb1:0xd7102A3744c302f167c53621453516345bC460d7) | [Tracking Sheet](https://docs.google.com/spreadsheets/d/1jLGkoiSjNr1UgG1spAH9ppBfrk_FqjJXg9CIw35nLCA/edit#gid=1325210165)                                                                                                                                                                |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:25]: Could consider https://discord.com/channels/405159462932971535/929890788551323678/1163935228205731920 for documents as "Partner Proposal Cheat Sheet" or smth

File: RPIPs/RPIP-36.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+
+## Motivation
+
+Currently, there is no overview of the active committees within Rocket Pool, or their purpose. Currently, users need to both know to look and know how to find the relevant RPIPs to find information that can be presented in a more user-friendly way. Further, historical membership is not being recorded in a structued way outside of the committees themselves.  
+
+## Committee Summary
+
+| Committee                             | Chief Mission                                                          | Date Established | Defining RPIPs                         | Multisig Addresses                                                                                                                                                                         | Public Documents                                                                                                                                                                                                                                                                         |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:23]: I don't think date established has much value here... info exists with earliest membership anyhow

File: RPIPs/RPIP-36.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Incentives Management Committee
+
+### Current Membership
+
+| Member Since | Members          | Member Type | Comment                                                                                                                                                                              | Address                                                                            |
+|--------------|------------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
+|   2023-09-22 | DrWorm           | Voting      | Selected as [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af) runner-up after Langers steps down. | [Address](https://etherscan.io/address/0xC701b5283822f84a2B358f79a4526FD6E1b8C395) |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:34]: Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.

File: RPIPs/RPIP-36.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+### Historic Membership
+
+| Date Changed | Members                                                                                                                    | Comment                                                                                                                                                                                     |
+|--------------|----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   2023-09-22 | Jasper, Valdorff, knoshua, Uisce, ramana, denum, NonFungibleYokem, phils.eth, DrWorm                                       | DrWorm selected as [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af) runner-up after Langers steps down. |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:48]: Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.

File: RPIPs/RPIP-36.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+|   2023-04-09 | phils.eth        | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x163aDf9feCB5307762F8bbc90EDeF0054d5d04fB) |
+|   2022-08-25 | Jasper           | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x04BEE613690e98A1959F236c38AbAa5f2439B14a) |
+|   2022-08-25 | Valdorff         | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x68DEFBfFffE6e05e58A01eb2A12dEf7151C83bE2) |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:40]: Treasurer?

File: RPIPs/RPIP-36.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+|   2023-11-10 | lookingforowls                   | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0xEEfE235EE65BceB660A1a7b19cfAB6e0Cf03AdA7) |
+|   2023-06-10 | rplmaxi.eth                      | Voting                    | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13) | [Address](https://etherscan.io/address/0x56eC9c988b16f81A18780C7eeE548edE60532eEE) |
+|   2023-06-10 | ShfRyn                           | Administrator, Non-Voting | [2023 Administrator Selection](https://dao.rocketpool.net/t/call-for-gmc-administrator-applications/1863/4)                          | [Address](https://etherscan.io/address/0x2bF5F18B21e78bAc8163ae8eF3B5Cd1530141570) |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:63]: Treasurer?

File: RPIPs/RPIP-36.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+|--------------|-----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   2023-11-10 | jcrtp, waqwaqattack, Ken, dondochaka, rplmaxi.eth, lookingforowls, FeelinggoodFeelingrt, epineph, rocknet | [2023 GMC Full Selection](https://vote.rocketpool.net/#/proposal/0xd0092d8d30e61ea3cdafdaeecbc81db8e84edbc045cf922365868de40f05ba13)                                                                  |
+|   2023-07-31 | jcrtp, waqwaqattack, Ken, mentor, objectObject, fornax, dondochaka, rplmaxi.eth, anisoptera.eth           | anisoptera.eth selected as [2022 GMC Expansion](https://vote.rocketpool.net/#/proposal/0x3431987750224c52ee4b1fd0510263ff05dbaee5b7258c078865042bffa03d8d) runner-up after Kevster steps down.        |
```
Comment (Valdorff) [inline RPIPs/RPIP-36.md:74]: Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.

a couple more instances in this table

**Comment by Valdorff** [comment]
Generally love it. Left some comments. Would also like to consider if this should be a superset to include RPIP Editors and the pDAO treasurer. They are not MC members, but tracking could be nice?

File: RPIPs/RPIP-36.md (hunk 2 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-36.md:23]: Eh, it's vaguely interesting and I don't think it does any harm. I might reorder it so the date column is leftmost.

File: RPIPs/RPIP-36.md (hunk 3 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-36.md:34]: Agree

File: RPIPs/RPIP-36.md (hunk 1 above)
Comment (LongForWisdom) [reply RPIPs/RPIP-36.md]: Decided not, will add if it moves out of discord.

**Comment by LongForWisdom** [comment]
Updated re Val's comments.

**Pull request merged**